### DISCUSSION TITLE
Alternative for Selenium CDP Tool for Appium

### DISCUSSION THREAD

**Discussion by iamgauravkhanna** [appium/appium#19609] [General]
Hello Everyone,

I am working on a task wherein I need to capture and validate particular post call.

For Website, I am able to do so smoothly with help of Selenium CDP Tool. This works well on dockerized setup.

I am trying to do same thing in Android/iOS Native App. I am able to capture request via MitmProxy but this solution will work if complete setup is present on local machine. 

Now challenge has posed when same needs to be done remotely. 

Kindly share your thoughts.

**Comment by jlipps** [comment]
Unfortunately, something like MitmProxy is the way to do this.

**Comment by sudharsan-selvaraj** [comment]
@iamgauravkhanna We have recently shipped a new lighweight [plugin](https://github.com/AppiumTestDistribution/appium-interceptor-plugin) for appium 2.0 which is similar to mitmproxy but everything is handled in server side. Currently we support api mocking. Do check it out and for any new feature request feel free to raise an issue and we will be happy help you out.

**Reply by iamgauravkhanna** [reply]
Thanks @sudharsan-selvaraj I will explore this option.

**Comment by thanggggg1** [comment]
Do you have any idea for this @iamgauravkhanna
